app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,5 @@
 class PostsController < ApplicationController
-  def index
-    render plain: 'Here is a list of posts'
-  end
```

Comment:
- [ ] In the migration file that creates the Post, you created the author_id twice. One was by explicitly stating the column and the other was by referencing the `users` table. I would advise you to remove the one that explicitly stated and then remove the statement that creates an index on the author_id since the reference statement will do that automatically. The reason why I asked for this change is that when I run the migration file, I get the error that author_id has already been created. 👍 💻 

https://github.com/lengushuru/Blog-app-on-rails/blob/1f5f1cbaccabedbbb6a34a63d181570c2225086d/db/migrate/20230621193212_create_posts.rb#L1-L17

#### Hint: Remove the following statement from the files:

```ruby
t.integer :author_id
 add_index :posts, :author_id
```

### Error Output

![image](https://github.com/lengushuru/Blog-app-on-rails/assets/77106962/bb1e8fb5-a4fa-4f1f-83b1-3871b7806fc1)